### PR TITLE
Minor fix to ceph join script, get doc to match script (CASMTRIAGE-4712)

### PR DESCRIPTION
**Pull request by bklei**
# Description

Fix for https://jira-pro.its.hpecorp.net:8443/browse/CASMTRIAGE-4712 -- fix pdsh command issue, make script match doc.  Ran on fanta.

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by mitchty — APPROVED** [review]

**Review by med-hpe — APPROVED** [review]
LGTM

File: scripts/join_ceph_cluster.sh [hunk 1]
```diff
@@ -27,205 +27,66 @@
...
-  fi
-  WAS_OSD=$(ssh $node ceph node ls|jq -r --arg h $host 'any(.osd|keys; .[] == $h)')
-  if [[ "$WAS_OSD" != "true" ]]; then 
-    WAS_OSD="false"
-  fi
-  if [[ "$WAS_OSD" == "true" ]]
-  then
-    #shellcheck disable=SC2207
-    OSDS+=($(ssh $node "ceph osd ls-tree $host"))
+for node in ncn-s001 ncn-s002 ncn-s003; do
```
[the remark this answers is not in the document]
Comment (rustydb) [reply scripts/join_ceph_cluster.sh:32]: I think we ever only have 3.. @bklei might know.

[the remark this answers is not in the document]
Comment (bklei) [reply scripts/join_ceph_cluster.sh:32]: It's a good question -- we certainly do have osds on storage node beyond the first three.  but i'm not sure that's what this for loop is targeting.  It looks like this is for 'mon' nodes (which are only on the first three).  @leliasen-hpe does that sound correct?

File: operations/node_management/Rebuild_NCNs/Re-add_Storage_Node_to_Ceph.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ Use the following procedure to re-add a Ceph node to the Ceph cluster.
 
    host=$(hostname)
 
-   > ~/.ssh/known_hosts
+   true > ~/.ssh/known_hosts
```
[the remark this answers is not in the document]
Comment (bklei) [reply operations/node_management/Rebuild_NCNs/Re-add_Storage_Node_to_Ceph.md:20]: good point, may pivot to install with subsequent PRs (assuming functionally the current change works -- it does appear elsewhere and is the recommended lint remediation)

**Pull request closed without merging**